Celeste.Mod.mm/Patches/FireBall.cs

```diff
@@ -15,6 +21,10 @@ public patch_FireBall(Vector2[] nodes, int amount, int index, float offset, floa
             // no-op. MonoMod ignores this - we only need this to make the compiler shut up.
         }
 
+        [MonoModIgnore]
+        [PatchModSpawnEntityData] // handled in MonoModRules.Mod.cs
+        public override extern void Added(Scene scene);
```

Comment:
does this need there are new requirements on how you initialize entities? :cold_sweat:

Reply:
Not really?
In effect, all this Patch does is, for vanilla entities where one entity spawns a series of other entities, the EntityData from the one entity is passed to all the other entities